plugins/feeds/public/benkow_rat.py

```diff
@@ -0,0 +1,61 @@
+import requests
+from datetime import datetime, timedelta
+import csv
+import logging
+
+from core.observables import Url, Ip
+from core.feed import Feed
+from core.errors import ObservableValidationError
+from core.config.config import yeti_config
+
+class BenkowTrackerRat(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=12),
+        "name": "BenkowTrackerRat",
+        "source" : "http://benkow.cc/export_rat.php",
+        "description": "This feed contains known Malware C2 servers",
+    }
+
+    def update(self):
+        resp = requests.get(self.source, proxies=yeti_config.proxy)
+        if resp.ok:
```

Comment:
It's better to use

```
if not resp.ok:
  raise GenericYetiError
```

So that any problems with the upstream will be reflected in the web UI